### ISSUE TITLE
MPM example

### ISSUE THREAD

**Issue by tiborauer** [bids-standard/bep001#34]
Regarding the examples, the current version contains a template
`sub-<participant_label>[_ses-<session_label>][_<indexable_metadata>-<index>][_acq-<label>][_part-<mag/phase>][_ce-<label>][_rec-<label>][_run-<index>]_<suffix>.nii.gz`,
so the MPM example should be:
- `sub-01_echo-1_acq-MTon_part-mag_MPM.nii.gz`
- `sub-01_echo-1_acq-MToff_part-mag_MPM.nii.gz`
- `sub-01_echo-1_acq-T1w_part-mag_MPM.nii.gz`
(i.e. "acq" before "part")

**Comment by agahkarakuzu** [comment]
Please see [this branch](https://github.com/bids-standard/bep001/tree/MPM_example), where I changed file names. Remaining steps: 

- [ ] Collapse `sub-01_acq-MToff_MPM`, `sub-01_acq-MTon_MPM` and `sub-01_acq-T1w_MPM` json files into one file called `sub-01_MPM.json`. 
- [ ] Edit json files with respect to the changes.

**Comment by tiborauer** [comment]
AFAICS, the images in the [MPM folder](https://github.com/bids-standard/bep001/tree/MPM_example/examples/ds-04/sub-01/anat) still follows the previous order:
E.g.:
- `sub-01_echo-1_part-mag_acq-MToff_MPM.nii.gz` (i.e. "part" before "acq")

**Comment by agahkarakuzu** [comment]
@tiborauer thanks for the heads up, you're right. I will correct them as I edit json files. 
```
        sub-<participant_label>[_ses-<session_label>][_<indexable_metadata>-<index>][_acq-<label>][_part-<mag/phase>][_ce-<label>][_rec-<label>][_run-<index>]_<suffix>.nii[.gz]
```

**Comment by lazaral** [comment]
@agahkarakuzu thank you for nice fixes on the MPM example! 

I'll start working on that branch as well to create the 'derivatives' folder - happy to help with the json files as well if useful, or I can give it a quick proof-read afterwards. I had a quick look now and noticed a small spelling mistake in the third PDw echo json where it says 'of' rather than 'off', but it's probably not worth correcting now as it will get changed again?

**Comment by agahkarakuzu** [comment]
> I'll start working on that branch as well to create the 'derivatives' folder

Thanks for #38 :) 

> happy to help with the json files as well if useful, or I can give it a quick proof-read afterwards. 

Thank you, feel free to work on it as you like! 

Regarding json files, the root json file (`sub-01_suffix.json`) should be containing some extra metadata fields (e.g. vendor and sequence specific tags). At least, this was the idea in the gdoc. Should we include this in the example datasets as well? @Gilles86 @KirstieJane maybe we can discuss this in the next meeting.

**Comment by KirstieJane** [comment]
Hey folks - in #22 I think we're not following the derivatives proposal accurately.... (Here's the proposal: https://github.com/bids-standard/bids-specification/blob/derivatives/src/05-derivatives/01-introduction.md)

We should have a pipeline name for each of the derived folders, right?

I might be totally missing the mark here.... 😬 

UPDATE - I've made an `hmri` pipeline folder in the PR to @lazaral's branch (https://github.com/lazaral/bep001/pull/1)

**Comment by KirstieJane** [comment]
This is related to #46

**Comment by lazaral** [comment]
I fixed some of these issues in  #38 - hopefully it is fairly up to date now, let me know if there's anything more I should change.
